src/otx/core/ov/ops/infrastructures.py

```diff
@@ -233,6 +233,8 @@ def from_ov(cls, ov_op):
             if not np.array_equal(data, data_):
                 logger.warning(f"Overflow detected in {op_name}")
             data = torch.from_numpy(data_)
+        elif data.dtype == np.uint16:
```

Comment:
@sovrasov I have little concern about this changes to resolving https://github.com/openvinotoolkit/training_extensions/issues/2675 because it caused by updating the default behavior of the model optimizer (mo) in the openvino 2023.1 (turned on as true to "--compress_to_fp16" option). I think the intention to the core module's unitests seemed to expecting uncompressed model as input but this change will not fix this thing.
```suggestion
        elif data.dtype == np.uint16:
```

Reply:
Do you mean that with these changes some other tests are expected to fail?

Reply:
no. we can check that all tests were passed with you change but what i meant is that the data type 'unintXX' is the artifact of the compression by model optimizer. so the way that i expect is that should be omitting compression option from the graph generation flow.
but as i've mentioned before, i don't have enough idea about the things in the otx.core.ov modules so if you think this is the correct changes, you can merge it. these comments from me are only my point of view to see the root cause of this issue.

Reply:
Yeah, I see your point. In my opinion, dtype compatibility problem is not a big deal, since we can always substitute a wider type when building a torch model.
I discovered one more failing test, and it raises even a bigger concern: graph patterns that were hardcoded in the module are not valid for the newly generated IRs. I'll skip one more test, because graph transformation carried out there is no longer applicable.